bin/confidence

```diff
@@ -30,8 +31,8 @@ internals.argv = Yargs.usage('Usage: $0 -c config.json [--filter.criterion=value
 
 internals.getConfig = function () {
 
-    var config = null;
-    var configPath = internals.argv.c[0] !== '/' ? process.cwd() + '/' + internals.argv.c : internals.argv.c;
+    let config = null;
+    const configPath = internals.argv.c[0] !== '/' ? process.cwd() + '/' + internals.argv.c : internals.argv.c;
```

Comment:
Could make use of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith